### PR TITLE
Fix domain translation

### PR DESCRIPTION
**Pull request by lartist**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Fix domain translation
| Type?             | bug fix
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | auto tests 🟢 https://github.com/lartist/ga.tests.ui.pr/actions/runs/4697213600
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

**Review by FabienPapet — DISMISSED** [review]

File: classes/lang/KeysReference/FeatureFlagLang.php [hunk 1]
```diff
@@ -65,7 +65,7 @@
 trans('Enable or disable the migrated countries page.', 'Admin.Advparameters.Help');
 
 // States feature flag
-trans('States', 'Admin.Advaprameters.Feature');
+trans('States', 'Admin.Advprameters.Feature');
```
Comment (boherm) [inline classes/lang/KeysReference/FeatureFlagLang.php]:
```suggestion
trans('States', 'Admin.Advparameters.Feature');
```
I think :-D

Comment (jolelievre) [reply classes/lang/KeysReference/FeatureFlagLang.php]: Indeed

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: classes/lang/KeysReference/FeatureFlagLang.php (hunk 1 above)
Comment (lartist) [reply classes/lang/KeysReference/FeatureFlagLang.php]: Thanks @boherm. Done

**Review by nicosomb — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @lartist

**Pull request merged**